src/explorer/DeploymentSlotsTreeItem.ts

```diff
@@ -75,19 +74,20 @@ export class DeploymentSlotsTreeItem implements IAzureParentTreeItem {
         return new DeploymentSlotTreeItem(newSite);
     }
 
-    private async promptForSlotName(slotLabels: string[]): Promise<string | undefined> {
+    private async promptForSlotName(node: IAzureParentNode<DeploymentSlotsTreeItem>): Promise<string | undefined> {
         return await window.showInputBox({
             prompt: 'Enter a unique name for the new deployment slot',
             ignoreFocusOut: true,
-            validateInput: (value: string) => {
+            validateInput: async (value: string) => {

```

Comment:
Do you need to update the vscode engines value to the latest one in package.json in order to use async validation?

Reply:
No, I don't think so.  I tested it with npm update and with the package.json that is currently in the repo and both seem to work correctly.